doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -489,7 +489,14 @@ Modem libraries
 
 * :ref:`pdn_readme` library:
 
-  * Updated the library to use ePCO mode if the Kconfig option :kconfig:option:`CONFIG_PDN_LEGACY_PCO` is not enabled.
+  * Updated:
+
+    * Updated the library to use ePCO mode if the Kconfig option :kconfig:option:`CONFIG_PDN_LEGACY_PCO` is not enabled.
```

Comment:
```suggestion
  * Updated the library to use ePCO mode if the Kconfig option :kconfig:option:`CONFIG_PDN_LEGACY_PCO` is not enabled.
```
Since we just have a single entry.